src/BloomBrowserUI/bookEdit/readerSetup/ReaderSetup.jade

```diff
@@ -1,189 +0,0 @@
-doctype html
```

Comment:
found it. put it back. one of the merges didn't handle a capitalization change well... :(